src/UserService.Mappers/RequestsMappers/UserRequestMapper.cs

```diff
@@ -27,7 +28,14 @@ public DbUser Map(UserRequest value)
                 Status = value.Status,
                 AvatarFileId = value.AvatarFileId,
                 IsActive = value.IsActive,
-                IsAdmin = value.IsAdmin
+                IsAdmin = value.IsAdmin,
+                Connections = value.Connection?.Select(x => new DbConnection
```

Comment:
just a question: why is here  "?" ?

Reply:
Because here we are checking the conditions of the object(it shouldn't be null)